salt/metalk8s/orchestrate/solutions/import-components.sls

```diff
@@ -0,0 +1,50 @@
+{%- set bootstrap_id = pillar.bootstrap_id %}
+{%- set version = pillar.metalk8s.nodes[bootstrap_id].version %}
+
+Import the Solutions archives:
+  salt.state:
+    - tgt: {{ bootstrap_id }}
+    - saltenv: metalk8s-{{ version }}
+    - sls:
+      - metalk8s.solutions.available
+
+Refresh master pillar:
+  module.run:
+    - saltutil.refresh_pillar: []
+    - require:
+      - salt: Import the Solutions archives
+
+Wait for master pillar to be refreshed:
```

Comment:
Isn't `refresh_pillar` blocking?

Reply:
No, it's not.

Reply:
Can we add a mention about this in #650 then? To not forget about this ugly 10 seconds sleep...

Reply:
Changed the code and removed this as it's no longer needed.